node-notes-cli/mumboJumbo.ts

```diff
@@ -0,0 +1,71 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+type Data = {
+  nextId: number;
+  notes: Record<string, string>;
+};
+
+const [, , command, string1, string2] = process.argv;
+
+try {
+  switch (command) {
+    case 'read': {
+      const data = await test();
+      for (const key in data.notes) {
+        console.log(`${key} - ${data.notes[key]}`);
+      }
+      break;
+    }
+    case 'add': {
+      let objJson = await readFile('data.json', { encoding: 'utf-8' });
+      const objJsonParsed = JSON.parse(objJson);
+      objJsonParsed.notes[objJsonParsed.nextId.toString()] = string1;
+      objJsonParsed.nextId = objJsonParsed.nextId + 1;
+      objJson = JSON.stringify(objJsonParsed, null, 2);
+      await writeFile('data.json', objJson, { encoding: 'utf-8' });
+      break;
+    }
+    case 'delete': {
+      let objJson = await readFile('data.json', { encoding: 'utf-8' });
+      const objJsonParsed = JSON.parse(objJson);
+      for (const key in objJsonParsed.notes) {
+        switch (key) {
+          case string1: {
+            delete objJsonParsed.notes[string1];
+            break;
+          }
+          default:
+        }
+      }
+      objJson = JSON.stringify(objJsonParsed, null, 2);
+      await writeFile('data.json', objJson, { encoding: 'utf-8' });
+      break;
+    }
+    case 'update': {
+      let objJson = await readFile('data.json', { encoding: 'utf-8' });
+      const objJsonParsed = JSON.parse(objJson);
+      for (const key in objJsonParsed.notes) {
```

Comment:
There's no reason to do a loop for this. You can just use `objJsonParsed.notes[string1]` without the loop.